bin/alpen-cli/src/cmd/deposit.rs

```diff
@@ -111,7 +112,13 @@ fn prepare_deposit_request(
             .address
     };
 
-    let header_aux = DrtHeaderAux::new(recovery_public_key.serialize(), alpen_address.into());
+    let alpen_subject_bytes =
+        SubjectBytes::try_new(alpen_address.to_vec()).expect("must be valid subject bytes");
+    let header_aux = DrtHeaderAux::new(
+        recovery_public_key.serialize(),
+        AccountSerial::zero(),
```

Comment:
This deserves a comment that all accounts have `0` and reserved (special) accounts are `1.